package/MDAnalysis/coordinates/memory.py

```diff
@@ -0,0 +1,257 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
Yeah, sorry. As I mentioned in the comment right before yours, we were still in the process of merging in the newest changes, which is why MemoryReader still appeared. This is done now - so all MemoryReader changes should now be gone.